flatnav/util/SimdBaseTypes.h

```diff
@@ -56,7 +56,7 @@ struct simd128bit {
 #endif
   }
 
-  bool operator=(const simd128bit &other) {
+  bool operator==(const simd128bit &other) {
```

Comment:
Here assigning values like `diff = v1 - v2` while computing distances was using the wrong overloaded operator.

Reply:
Specifically this line: https://github.com/BlaiseMuhirwa/flatnav-experimental/blob/5dbc3ef6a1096cca4ba3d2f81ebc4a4c7aa2c0bf/flatnav/util/SquaredL2SimdExtensions.h#L113